packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -785,7 +785,9 @@ def _sender_act(self) -> Generator[None, None, None]:
 
             tx_hashes_history = self.synchronized_data.tx_hashes_history
             if tx_data["tx_digest"] != "":
-                tx_hashes_history.append(cast(str, tx_data["tx_digest"]))
+                tx_hashes_history.append(
+                    cast(str, tx_data["tx_digest"])
+                )  # pragma: no cover
```

Comment:
to painful to mock

Reply:
We need a test for this. Discuss with @Adamantios